calico_containers/calico_ctl/node.py

```diff
@@ -393,6 +397,22 @@ def _start_libnetwork_container(etcd_authority, libnetwork_image):
     print "Calico libnetwork driver is running with id: %s" % cid
 
 
+def _setup_ip_forwarding():
+    """
+    Ensure that IP forwarding is enabled.
+    :return: None
+    """
+    # Enable IP forwarding since all compute hosts are vRouters.
+    # IPv4 forwarding should be enabled already by docker.
+    if "1" not in sysctl("net.ipv4.ip_forward"):
```

Comment:
I don't think there's any need to read this then write it. We should just write it every time.

And rather than writing using sysctl, let just use Python

with open('/proc/sys/net/ipv4/ip_forward', 'w') as f:
   f.write("1")

Then we can get rid of one more use of the sh module.